tests/config/test_git.py

```diff
@@ -0,0 +1,15 @@
+import os
+from unittest import TestCase
+from unittest.mock import patch
+
+from nestor_api.config.git import GitConfiguration
+
+
+# pylint: disable=too-few-public-methods
```

Comment:
Maybe we could add this rule to the `.pylintrc` of the tests?

Reply:
I don't think it should be, actually the rule is right, having a class with only one method would makes no sense (better to do a simple function) but the thing is that here, we know it will probably evolve with other methods :)